### ISSUE TITLE
Display related events divided into multiple pages

### ISSUE THREAD

**Issue by ssotoya** [MISP/MISP#2687]
Currently, all related events are displayed on the event view screen.
In this state, if you try to open an event with the number of related events from thousands to tens of thousands in the browser, the browser crashes.
(Even if it does not crash, it takes a huge amount of time to be displayed)

 Currently we want to accumulate information such as OSINT etc. to MISP in large quantity (about millions of events) and want to see the correlations. Then, a large number (thousands to tens of thousands) of related events will occur. As mentioned above, trying to open the event view screen in this state may cause the browser to crash, or it takes a huge amount of time, which is not practical.

 As for the list display of this related events, I would like to add a function to split the page like attributes. This makes it possible to quickly check the necessary information even if there are many related events.
Also, in conjunction with this case, I believe that the following measures will be more effective.
[Enhancement] Show Related Events Count within a MISP Event Issue #2650 https://github.com/MISP/MISP/issues/2650

**Comment by iglocska** [comment]
Yeah we definitely need to deal with this. Initially highly correlating data sets were not envisioned but we have it popping up in more and more communities. Will look into this. 

We have similar functionality for feed correlations already, where the full list is only shown on demand.

**Comment by ssotoya** [comment]
Thank you for your quick reply. I will cooperate if I can do it. Thank you.
